sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.aest.controller.assessOfferingChangeRequest.e2e-spec.ts

```diff
@@ -305,6 +308,71 @@ describe("EducationProgramOfferingAESTController(e2e)-assessOfferingChangeReques
     },
   );
 
+  it(
+    "Should determine in-progress applications for a requested offering when the offering change is approved" +
+      "and there are applications associated with the requested offering.",
+    async () => {
+      // Arrange
+      const applications: Application[] = [];
+      for (let i = 0; i < 2; i++) {
+        const application = await saveFakeApplication(
+          db.dataSource,
+          {
+            institution: collegeFLocation.institution,
+            institutionLocation: collegeFLocation,
+          },
+          {
+            applicationStatus: ApplicationStatus.InProgress,
+          },
+        );
+        application.currentAssessment.offering = precedingOffering;
+        await db.application.save(application);
+        applications.push(application);
+      }
```

Comment:
I am not sure if creating an array of applications adds value to the test.

I would say either create an application in `In-Progress` (or `Assessment` or `Enrolment Status`) and one Completed application both.